sklearn/tests/test_cross_validation.py

```diff
@@ -25,6 +25,12 @@
 with warnings.catch_warnings():
     warnings.simplefilter('ignore')
     from sklearn import cross_validation as cval
+    try:
+        # We should be able to import this from cross_validation
+        from sklearn.cross_validation import FitFailedWarning as FFW_
```

Comment:
Should we do away with this try except? I don't see the reason behind catching an error and raising another.